make-rules/ips.mk

```diff
@@ -194,12 +194,15 @@ PUBLISH_STAMP ?= $(BUILD_DIR)/.published-$(MACH)
 
 # Do all that is needed to ensure the package is consistent for publishing,
 # except actually pushing to a repo, separately from the push to the repo.
-pre-publish:	build install $(PRE_PUBLISH_STAMP)
+# Note that running "install" rule as a dependency for multiple targets can
+# have adverse effects, so instead we call it at most once by requiring the
+# touched ALL_INSTALLED_STAMP file.
+pre-publish:	$(ALL_INSTALLED_STAMP) $(PRE_PUBLISH_STAMP)
```

Comment:
it dependent on build install and now only on install (through ALL_INSTALLED_STAMP). Can it cause any issues?